riva/clients/asr/riva_asr_client.cc

```diff
@@ -452,6 +470,9 @@ main(int argc, char** argv)
   str_usage << "           --stop_history_eou=<int>" << std::endl;
   str_usage << "           --stop_threshold=<float>" << std::endl;
   str_usage << "           --stop_threshold_eou=<float>" << std::endl;
+  str_usage << "           --source_language=<string>" << std::endl;
+  str_usage << "           --target_language=<string>" << std::endl;
+  str_usage << "           --task=<string>" << std::endl;
```

Comment:
we should support these command line params through the generic `--metadata` field we have in the client. Adding dedicated translation specific params (source/target language, task) can be confusing for users in the ASR client.

Reply:
Updated to use custom_configuration